Source/Globals/UIGlobals.h

```diff
@@ -36,3 +36,10 @@ extern NSString *Elision_;
 extern NSString *Error_;
 extern NSString *Warning_;
 extern const NSString *UI_;
+
+// Swipe actions
+static bool DismissVCAfterProgress_;
+static bool DismissVCAsQueue_;
+static bool FromSwipeAction_;
+static bool ClickDowngrade_;
+static bool ClickBuy_;
```

Comment:
We're trying to refactor and remove globals right now, could you add these as instance properties?